tests/test_compile_models.py

```diff
@@ -0,0 +1,203 @@
+    )
+    compile_models.main()
+
+    stubber_client.assert_no_pending_responses()
+
+    with open(tmp_path / "assessments.csv", 'r') as f:
+        file = f.read()
+        assert file == \
+            'name,id\n' +\
+            '"Unit 1, Section A Quiz",0\n' +\
+            '"Unit 1, Section B Quiz",1\n' +\
+            '"Unit 1, Section C Quiz",2\n' +\
+            'Unit 1 STAAR Review,3\n' +\
+            'Unit 1 Quiz,4\n' +\
+            '"Unit 2, Section A Quiz",5\n' +\
+            '"Unit 2, Section B Quiz",6\n' +\
+            '"Unit 2, Section C Quiz",7\n' +\
+            '"Unit 2, Section D Quiz",8\n' +\
+            'Unit 2 STAAR Review,9\n' +\
+            'Unit 2 Quiz,10\n'
+
+    with open(tmp_path / "users.csv", 'r') as f:
+        file = f.read()
+        assert file == \
+            'uuid,first_name,last_name,email\n' +\
+            'cd613e30-d8f1-6adf-91b7-584a2265b1f5,' +\
+            'David,Tucci,davidtucci13431@gmail.us\n' +\
+            '1e2feb89-414c-343c-1027-c4d1c386bbc4,' +\
+            'Deann,James,deannjames121231@gmail.com\n' +\
+            '78e51061-7311-d8a3-c2ce-6f447ed4d57b,Don,Castle,' +\
+            'doncastle1234123@gmail.com\n' +\
+            '35bf992d-c9e9-c616-612e-7696a6cecc1b,James,Stewart,' +\
+            'jamesstewart1234121@gmail.com\n' +\
+            'e4b06ce6-0741-c7a8-7ce4-2c8218072e8c,Marguerite,Bell,' +\
+            'margueritebell53245@gmail.com\n'
+
+    with open(tmp_path / "enrollments.csv", 'r') as f:
+        file = f.read()
+        assert file == \
+            'user_uuid,course_id,role\n' +\
+            'cd613e30-d8f1-6adf-91b7-584a2265b1f5,1,student\n' +\
+            '1e2feb89-414c-343c-1027-c4d1c386bbc4,1,student\n' +\
+            '78e51061-7311-d8a3-c2ce-6f447ed4d57b,1,teacher\n' +\
+            '35bf992d-c9e9-c616-612e-7696a6cecc1b,1,student\n' +\
+            'e4b06ce6-0741-c7a8-7ce4-2c8218072e8c,1,student\n'
+
+    with open(tmp_path / "oneroster_demographics.csv", 'r') as f:
+        file = f.read()
+        assert file == \
+            'user_uuid,birth_date,sex,american_indian_or_alaska_native,' +\
+            'asian,black_or_african_american,' +\
+            'native_hawaiian_or_other_pacific_islander,' +\
+            'white,demographic_race_two_or_more_races,' +\
+            'hispanic_or_latino_ethnicity\n' +\
+            'cd613e30-d8f1-6adf-91b7-584a2265b1f5,2007-11-29,' +\
+            'male,false,false,false,false,true,false,false\n' +\
+            '1e2feb89-414c-343c-1027-c4d1c386bbc4,2008-03-17,' +\
+            'female,false,false,false,false,false,false,true\n' +\
+            '35bf992d-c9e9-c616-612e-7696a6cecc1b,2007-12-20,' +\
+            'male,false,false,true,false,false,false,false\n' +\
+            'e4b06ce6-0741-c7a8-7ce4-2c8218072e8c,2008-01-03,' +\
+            'female,true,false,false,false,false,false,false\n'
+
+    with open(tmp_path / "enrollments.csv", 'r') as f:
```

Comment:
I believe we are currently missing coverage / assertions on an expected `courses.csv` but we check `enrollments.csv` twice. Perhaps an oversight :man_shrugging: . 

I would suggest running the coverage report per the `README.md` locally to ensure any missed lines are expected.